recipes/fmi2/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.15)
+project(test_package LANGUAGES C CXX)
+
+find_package(fmi2 REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} PRIVATE fmi2::fmi2)
```

Comment:
Quick question, what's the minimum cppstd for these libraries?

Reply:
These are C headers, so I guess it does not apply.

Reply:
Oh, hadn't realized that, thanks for the extra info :)